utils/tzktHooks.ts

```diff
@@ -0,0 +1,152 @@
+import { useEffect, useState } from "react";
+import { TZKT_API_URL } from "../context/config";
+
+/**
+ * Returns the balance of the given address
+ * @param address tz/kt Tezos address
+ */
+export const useTzktBalance = (address: string | null) => {
+  const [balance, setBalance] = useState(0);
+  useEffect(() => {
+    if (address == null) return;
+    (async () => {
+      const response = await fetch(
+        `${TZKT_API_URL}/v1/accounts/${address}/balance`
+      );
+
+      if (response.status == 200) {
```

Comment:
`===`

Reply:
fixed in: https://github.com/marigold-dev/tzsafe-ui/pull/167/commits/7624e35d0a26f68d2cdc0ca77342b0fc590bbe60